src/vstart.sh

```diff
@@ -202,13 +202,12 @@ usage=$usage"\t-s, --standby_mds: Generate standby-replay MDS for each active\n"
 usage=$usage"\t-l, --localhost: use localhost instead of hostname\n"
 usage=$usage"\t-i <ip>: bind to specific ip\n"
 usage=$usage"\t-n, --new\n"
-usage=$usage"\t-N, --not-new: reuse existing cluster config (default)\n"
 usage=$usage"\t--valgrind[_{osd,mds,mon,rgw}] 'toolname args...'\n"
 usage=$usage"\t--nodaemon: use ceph-run as wrapper for mon/osd/mds\n"
 usage=$usage"\t--redirect-output: only useful with nodaemon, directs output to log file\n"
 usage=$usage"\t--smallmds: limit mds cache size\n"
 usage=$usage"\t-m ip:port\t\tspecify monitor address\n"
-usage=$usage"\t-k keep old configuration files\n"
+usage=$usage"\t-k keep old configuration files (default)\n"
```

Comment:
`default` is a change in behavior. Why?

Reply:
No, It was the same behavior without this patch as well. When '-n' was not passed, '-N' was being  used which used the existing configuration files if present. In the same way, now if -n is not passed, it uses existing configuration files. Hence mentioned as default.

Reply:
yeah. before this change, "new" is 0, and "overwrite_conf" is 1 by default. but later, "overwrite_conf" is reset if  "new" is "0". so, by default, the existing conf is kept intact.

Reply:
@batrick does this make sense with the explanations above?

Reply:
Yes thanks.